quickstarts/tidbcloud/config.yml

```diff
@@ -0,0 +1,55 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: tidbcloud
+
+# Displayed in the UI (required)
+title: TiDB Cloud
+
+# Long-form description of the quickstart (required)
+description: |-
+  ## Quickstart for TiDB Cloud monitoring
+  TiDB Cloud is a fully-managed Database-as-a-Service (DBaaS) solution for TiDB, an open-source, distributed SQL database that is designed to handle both online transaction processing (OLTP) and online analytical processing (OLAP) workloads.
+  
+  ### New Relic TiDB Cloud quickstart features
+  With monitoring statistics enabled, you can observe key metrics to analyze the performance of your cluster. 
+  
+  ### TiDBCloud Quickstart instructions
+  Check out [Metrics available to New Relic](https://docs.pingcap.com/tidbcloud/monitor-newrelic-integration#metrics-available-to-newrelic) to learn more about New Relic monitoring for TiDBCloud.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+    New Relic's TiDB Cloud quickstart provides performance insights including database time, queries per second, and resource utilization. Install the quickstart to better understand utilization of resources and monitor performance issues.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - PingCAP
+  - lilyjazz
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - tidb
+  - tidb cloud
+  - cloud
+  - dbaas
+  - htap
+  - serverless
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - tidbcloud-integration
+
+dashboards:
+  - TiDB Cloud Monitoring
```

Comment:
this should reflect the directory name under the dashboards/ directory, so "tidbcloud"

Reply:
done